### ISSUE TITLE
`squeeze` parameter for python `Sum` layer does not work

### ISSUE THREAD

**Issue by dding3** [intel-analytics/analytics-zoo#1208]
`squeeze` has been set to `True` when using `Sum` layer, however, the sum dimension is not squeezed. It can be reproed by following code:
`from zoo.pipeline.api.keras.models import Sequential
embedding = Sequential()
embedding.add(Reshape([3 * 2], input_shape=(3, 2)))\
    .add(Embedding(15, 10, input_length=3 * 2))\
    .add(Dropout(0.1))\
    .add(Reshape((3, 2, 10)))
embedding.add(KerasLayerWrapper(Sum(dimension=3, squeeze=True)))`
